VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+
+### Do validators need to be publicly identified?
+
+No, they don’t. Each delegator will value each validator based on objective and subjective criteria. Validators will be able (and are recommended) to register a website address when they nominate themselves so that they can advertise their operation as they see fit. Some delegators may prefer a website that clearly displays the team running the validator and their resume, while others might prefer anonymous validators with positive track records. Most likely both identified and anonymous validators will coexist in the validator set.
+
+### What are the directives of a validator?
+
+Validators have two main directives:
+* **Be able to constantly run a correct version of the software:** validators need to make sure that their servers are always online and their private keys are not compromised. 
+* **Actively participate in governance:** validators are required to vote on every proposal.
+
+Additionally, validators are expected to be active members of the community. They should always be up to date with the current state of the ecosystem so that they can easily adapt to any change.
+
+### What does ‘participate in governance’ entail?
+
+Validators and delegators on the Cosmos Hub can vote on proposals to change preset parameters of the system automatically (such as the block gas limit), coordinate upgrades, as well as vote on amendments to the human-readable constitution that govern the policies of the Cosmos Hub. The governance system allows for smooth coordination within the network. 
+
+Validators play a special role in the governance system. Being the pillars of the system, they are required to vote on every proposal. It is especially important since delegators who do not vote will inherit the vote of their validators. If a validator does not vote on a proposal it will get a small punishment, and if it does not vote on many successive proposals, the validator will become unbonded.
+
+### What does staking imply?
+
+Bonded ATOM holders receive rewards and the right to participate in governance. In exchange, they accept to lock their funds. If they want to retrieve them, they need to wait for a *three weeks unbonding period*.
+
+If a validator misbehaves, a certain portion of its total stake is slashed (the severity of the penalty depends on the type of fault). This means that every user that bonded ATOMs to this validator gets penalized in proportion of its stake. Delegators are therefore incentivized to delegate to validators that they believe will behave correctly.
```

Comment:
However large validators that are run by exchanges might be willing to take the slashing all for themselves and reimburse users. :-)

Reply:
That's fine for small slashing (and this will need to out of band for now if it's to happen). However for the super large slashing... the validator may not have anything to reimburse with!

Reply:
@gamarin2

Reply:
addressed